modules/dr-restoring-cluster-state.adoc

```diff
@@ -554,6 +554,8 @@ etcd-ip-10-0-154-194.ec2.internal                2/2     Running     0
 etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0          9h
 ----
 
+To ensure that all workloads return to normal operation following a recovery procedure, you should restart each pod that stores Kubernetes API information. This includes {product-title} components such as routers, operators, and third-party components.
```

Comment:
```suggestion
To ensure that all workloads return to normal operation following a recovery procedure, restart each pod that stores Kubernetes API information. This includes {product-title} components such as routers, operators, and third-party components.
```
Per the IBM style guide, avoid using "should" in favor of being more direct.

Reply:
Thanks Mike.